.eslintrc.js

```diff
@@ -326,6 +326,8 @@ module.exports = {
     '/packages/amplify-category-custom/lib',
     '/packages/amplify-category-custom/resources',
     '/packages/amplify-category-custom/src/utils/generate-cfn-from-cdk.ts',
+    '/packages/amplify-graphql-types-generator/lib',
+    '/packages/amplify-graphql-docs-generator/lib',
```

Comment:
These packages have been removed from the project. I think you just need to delete them locally.